internal/api/attestationconfigapi/cli/main.go

```diff
@@ -56,29 +59,40 @@ func main() {
 
 // newRootCmd creates the root command.
 func newRootCmd() *cobra.Command {
-	rootCmd := &cobra.Command{
-		Use:   "COSIGN_PASSWORD=$CPW COSIGN_PRIVATE_KEY=$CKEY upload --version-file $FILE",
-		Short: "Upload a set of versions specific to the azure-sev-snp attestation variant to the config api.",
+	rootCmd := &cobra.Command{}
+	rootCmd.PersistentFlags().StringP("region", "r", awsRegion, "region of the targeted bucket.")
+	rootCmd.PersistentFlags().StringP("bucket", "b", awsBucket, "bucket targeted by all operations.")
+	rootCmd.PersistentFlags().Bool("testing", false, "upload to S3 test bucket.")
 
-		Long: fmt.Sprintf("The CLI uploads an observed version number specific to the azure-sev-snp attestation variant to a cache directory. The CLI then determines the lowest version within the cache-window present in the cache and writes that value to the config api if necessary. "+
+	rootCmd.AddCommand(newUploadCmd())
+	rootCmd.AddCommand(newDeleteCmd())
+
+	return rootCmd
+}
+
+func newUploadCmd() *cobra.Command {
+	uploadCmd := &cobra.Command{
+		Use:   "upload {azure|aws} {snp-report|guest-firmware} <path>",
```

Comment:
I have changed the interface here because the path is an argument for the action you want to take. Not an option (flag) that modifies the action. I also find it less confusing than having flags. I would ask others for their opinion. @elchead @msanft wdyt?

Reply:
While I see that deeply nested commands are uncommon, I think that it reads better as is atm. Since it's an internal tool I have no strong opinion and would honestly just keep as is.